src/core/server_status.proto

```diff
@@ -304,29 +304,72 @@ message SharedMemoryRegion
   //@@
   //@@  .. cpp:var:: string name
   //@@
-  //@@     The name for this shared memory region.
+  //@@     The name for this System/CUDA shared memory region in TRTIS.
   //@@
   string name = 1;
 
-  //@@  .. cpp:var:: string shared_memory_key
-  //@@
-  //@@     The name of the shared memory region that holds the input data
-  //@@     (or where the output data should be written).
-  //@@
-  string shared_memory_key = 2;
+  message SystemSharedMemoryInfo
+  {
+    //@@  .. cpp:var:: string shared_memory_key
+    //@@
+    //@@     The name of the shared memory region that holds the input data
+    //@@     (or where the output data should be written).
+    //@@
+    string shared_memory_key = 1;
+  }
+
+  message CudaSharedMemoryInfo
+  {
+    //@@  .. cpp:var:: string shared_memory_name
+    //@@
+    //@@     The name of the shared memory region in TRTIS that holds the
+    //@@     cudaIPC handle.
+    //@@
+    string shared_memory_name = 1;
```

Comment:
Is `shared_memory_name` here the same thing as `shared_memory_key` above?

Reply:
shared_memory_name (renamed to `handle_region_name`) is the name of the shared memory region registered in TRTIS that contains the same